tests/user_journey/test_happy_path_1.py

```diff
@@ -0,0 +1,197 @@
+    - User B (mentee) comments on the task
+    - User A (mentor) comments on the task
+    """
+
+    def setUp(self):
+        super().setUp()
+
+        self.mentor = UserModel(
+            name="Mentor A",
+            email=user1["email"],
+            username=user1["username"],
+            password=user1["password"],
+            terms_and_conditions_checked=user1["terms_and_conditions_checked"],
+        )  # User A
+
+        self.mentee = UserModel(
+            name="Mentee B",
+            email=user2["email"],
+            username=user2["username"],
+            password=user2["password"],
+            terms_and_conditions_checked=user2["terms_and_conditions_checked"],
+        )  # User B
+
+        self.mentor.is_email_verified = True
+        self.mentee.is_email_verified = True
+
+        self.mentor.available_to_mentor = True
+        self.mentee.need_mentoring = True
+
+        db.session.add(self.mentor)
+        db.session.add(self.mentee)
+        db.session.commit()
+
+        self.test_description = "A nice task description"
+
+    def test_happy_path_1(self):
+
+        mentor_auth_header = get_test_request_header(self.mentor.id)
+        mentee_auth_header = get_test_request_header(self.mentee.id)
+
+        # - User B (mentee) sends a request to User A (mentor)
+        # POST /mentorship_relation/send_request
+
+        request_body = {
+            "mentor_id": self.mentor.id,
+            "mentee_id": self.mentee.id,
+            "end_date": int((datetime.now() + timedelta(days=40)).timestamp()),
+            "notes": "some notes",
+        }
+        send_request_response = self.client.post(
+            "/mentorship_relation/send_request",
+            headers=mentee_auth_header,
+            content_type="application/json",
+            data=json.dumps(request_body),
+        )
+
+        self.assertEqual(201, send_request_response.status_code)
+
+        request_sent_response = self.client.get(
+            "/mentorship_relations",
+            headers=mentor_auth_header,
+            content_type="application/json",
+        )
+
+        requests = json.loads(request_sent_response.data)
+        mentorship_request = requests[0]
+        request_id = mentorship_request["id"]
+        request_state = mentorship_request["state"]
+
+        self.assertIsNotNone(request_id)
+        self.assertEqual(MentorshipRelationState.PENDING, request_state)
+
+        # - User A (mentor) accepts request
+        # POST /mentorship_relation/{request_id}/accept
+
+        accept_response = self.client.put(
+            f"/mentorship_relation/{request_id}/accept", headers=mentor_auth_header
+        )
+
+        self.assertEqual(200, accept_response.status_code)
+
+        mentee_current_relation = self.client.get(
+            f"/mentorship_relations/current", headers=mentor_auth_header
+        )
+        mentor_current_relation = self.client.get(
+            f"/mentorship_relations/current", headers=mentor_auth_header
```

Comment:
I assume this was intended to be `mentee_auth_header`?

Reply:
🙈  what an awesome catch! @SanketDG 👏 🎉